docs/widgets.md

```diff
@@ -20,7 +20,7 @@ Widgets define the data type and interface for entry fields. Netlify CMS comes w
 | `list`     | repeatable group of other widgets  | Immutable List of objects containing field values  |
 | `relation` | text input w/ suggestions dropdown | value of `valueField` in related entry (see below) |
 
-We’re always adding new widgets, and you can also [create your own](/docs/extending)!
+We’re always adding new widgets, and you can also [create your own](https://github.com/netlify/netlify-cms/blob/master/docs/extending)!
```

Comment:
We'll need to leave links as is for now, until we find a solution that doesn't negatively affect links on the docs site, which is generated from this repo.

There's some discussion about this in a recent PR - here's where we stand: https://github.com/netlify/netlify-cms/pull/471#issuecomment-312670664